scripts/solutions.sh

```diff
@@ -138,7 +138,12 @@ while :; do
     case $1 in
         -a|--archive)
             shift
-            ARCHIVES+=("$(readlink -f "$1")")
+            ARCHIVE_ABS_PATH=$(readlink -f "$1" ||:)
```

Comment:
What does `||:` do?

Reply:
`|| true` if you prefer (I can change it), it allows to ignore the non-zero exit code from readlink otherwise the script exits since we use `set -e`.

Reply:
Changed this for a clearer approach.